creator-node/src/services/stateMachineManager/stateMachineConstants.ts

```diff
@@ -92,7 +96,9 @@ export const MAX_QUEUE_RUNTIMES = Object.freeze({
   // Max millis to run a recurring sync job for before marking it as stalled
   RECURRING_SYNC: 6 /* min */ * 60 * 1000,
   // Max millis to run an update-replica-set job for before marking it as stalled
-  UPDATE_REPLICA_SET: 5 /* min */ * 60 * 1000
+  UPDATE_REPLICA_SET: 5 /* min */ * 60 * 1000,
+  // Max millis to run a recover-orphaned-data job for before marking it as stalled
+  RECOVER_ORPHANED_DATA: 60 /* min */ * 60 * 1000
```

Comment:
we want to keep all these jobs much shorter than 1hr right?

Reply:
I'm not exactly sure because I saw the DN query for finding users with the CN in their replica sometimes take minutes (and there will be many of these paginated queries). I should have a better idea once it's implemented and I can watch it run a few times on staging

Reply:
kk cool, i think we also have linear card to look into why those queries take so long